### PR TITLE
Add tests for ChatWindow

### PR DESCRIPTION
**Pull request by e1121208**

### CODE REVIEW

File: src/test/java/seedu/address/ui/ChatWindowTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        stage.show();
+    }
+
+    @BeforeEach
+    public void setup() throws Exception {
+        FxToolkit.registerStage(() -> new Stage());
+    }
+
+    @Test
+    public void testGetResponseGreetings() {
```
Comment (lrongyi) [inline src/test/java/seedu/address/ui/ChatWindowTest.java:36]: Note the incorrect naming convention of your tests

File: src/main/java/seedu/address/ui/ChatWindow.java [hunk 2]
```diff
@@ -79,7 +79,7 @@ public void handleSendButtonAction() {
      * @param message The user's input message.
      * @return The assistant's response as a String.
      */
-    private String getResponse(String message) {
+    public String getResponse(String message) {
         message = message.toLowerCase().trim();
 
         switch (message) {
```
Comment (lrongyi) [inline src/main/java/seedu/address/ui/ChatWindow.java:85]: Good idea for a chat window. Could possibly add cases where the user types in the COMMAND_WORD of each command and the chat returns the MESSAGE_USAGE

**Review by lrongyi — COMMENTED** [review]
Overall LGTM!!!

**Review by seandias — COMMENTED** [review]
Nice!

**Pull request merged**